docs/backend/json-files/fs/format.json

```diff
@@ -0,0 +1,4 @@
+{
+  "format": "fs",
+  "version": "1"
```

Comment:
We should move these JSON files into a single Markdown file. I can document the backend format.